changelog.d/20240122_102159_30907815+rjmello_retain_uep_config_exception.rst

```diff
@@ -0,0 +1,5 @@
+Changed
+^^^^^^^
+
+- Changed the default engine type to `GlobusComputeEngine`, which utilizes the
```

Comment:
Add a note that this is for newly configured endpoints?  As we aren't changing existing configs.

Reply:
Good call; better to be explicit.